client/sites/tools/staging-site/components/staging-site-card/card-content/manage-staging-site-card-content.tsx

```diff
@@ -80,6 +80,12 @@ const ActionButtons = styled.div( {
 	},
 } );
 
+const ExternalIcon = styled( Icon )( {
+	verticalAlign: 'middle',
```

Comment:
As we are using flex to style parent items, we could consider following the flex approach here, too, by setting alignment on parent span element.

Reply:
Thanks for feedback, Wojtek.

> As we are using flex to style parent items, we could consider following the flex approach here, too, by setting alignment on parent span element.

I have experimented with different approaches yesterday (including flex), but the results were not as expected on smaller screens where the staging site URL can get into two lines, e.g.:

![Markup on 2025-03-04 at 12:30:40](https://github.com/user-attachments/assets/58b72239-e373-415e-9ca9-ff4dba97a88a)